bin/ember

```diff
@@ -7,6 +7,11 @@ process.title = 'ember';
 var resolve = require('resolve');
 var exit = require('exit');
 
+var updateNotifier = require('update-notifier');
```

Comment:
we aim to rarely (if ever) make changes to this file, as it is shared between all (old/new) ember apps on a system.

This should likely be somwhere in lib/cli